### ISSUE TITLE
(Bug): support for Serpent 52

### ISSUE THREAD

**Issue by andika207** [Anarios/return-youtube-dislike#457]
### Browser

Serpent 52

### Browser Version

52

### Extension or Userscript?

Extension

### Extension/Userscript Version

2.1.0.3

### Video link where you see the problem

any youtube video

### What happened?

this addon doesn't work on Serpent 52 web browser http://rtfreesoft.blogspot.com/ which is a fork of Basilisk browser, with support for Windows XP powered computers though

<a href='https://postimg.cc/8FJD8Fx2' target='_blank'><img src='https://i.postimg.cc/s2K2h7mg/screenshot-2.png' border='0' alt='screenshot-2'/></a>

**Comment by sy-b** [comment]
- I'll consider it more as a feature request
- Currently the extension supports most known Firefox and Chromium based browsers. If Serpent 52 can get somehow get their extensions to work on it, then this extension should work on it. It seems to me that Serpent 52 supports Firefox extensions, so I think you should make sure that no other extension is conflicting with it.
- Try https://github.com/Anarios/return-youtube-dislike/wiki/Troubleshooting-Guide

**Comment by andika207** [comment]
> It seems to me that Serpent 52 supports Firefox extensions

it only supports some legacy extensions but they actually must be rewritten for this specific browser.

**Comment by sy-b** [comment]
>  but they actually must be rewritten for this specific browser

That makes it a feature request. I doubt the possibility of anyone rewriting the extension.

If you can get something like Tampermonkey to work on that browser, then you should be able to use the userscript. Give it a try.

**Comment by andika207** [comment]
@sy-b I'm done
this addon is neither compatible with this browser nor with the original Firefox 52
tried it out on my chrome 86 based browser and it's not compatible either @Anarios 
''Tampermonkey'' is not support int this browser

<a href='https://postimg.cc/f3J8QgxL' target='_blank'><img src='https://i.postimg.cc/CK4XD35j/Snap-Crab-No-Name-2022-5-8-22-56-11-No-00.png' border='0' alt='Snap-Crab-No-Name-2022-5-8-22-56-11-No-00'/></a>

<a href='https://postimg.cc/xkw3Djfx' target='_blank'><img src='https://i.postimg.cc/j55gfCGb/Snap-Crab-No-Name-2022-5-8-22-57-59-No-00.png' border='0' alt='Snap-Crab-No-Name-2022-5-8-22-57-59-No-00'/></a>

**Comment by andika207** [comment]
@sy-b I'm done
this addon is neither compatible with this browser nor with the original Firefox 52
tried it out on my chrome 86 based browser and it's not compatible either @Anarios 
''Tampermonkey'' is not supported int this browser

<a href='https://postimg.cc/f3J8QgxL' target='_blank'><img src='https://i.postimg.cc/CK4XD35j/Snap-Crab-No-Name-2022-5-8-22-56-11-No-00.png' border='0' alt='Snap-Crab-No-Name-2022-5-8-22-56-11-No-00'/></a>

<a href='https://postimg.cc/xkw3Djfx' target='_blank'><img src='https://i.postimg.cc/j55gfCGb/Snap-Crab-No-Name-2022-5-8-22-57-59-No-00.png' border='0' alt='Snap-Crab-No-Name-2022-5-8-22-57-59-No-00'/></a>

**Comment by sy-b** [comment]
> this addon is neither compatible with this browser nor with the original Firefox 52

yeah, the browser API features that RYD is using are probably not supported by older browsers 😅

If older extension versions worked, you can continue using them. They should work as long as the API response remains the same.

Another option - 
Try this - [KellyC Show YouTube Dislikes](https://chrome.google.com/webstore/detail/kellyc-show-youtube-disli/gmdihkflccbodfkfioifolcijgahdgaf)

**Comment by andika207** [comment]
@sy-b neither supported... looks like the ruzzians are allergic to some web browsers. :D

<a href='https://postimg.cc/MMsFCyRx' target='_blank'><img src='https://i.postimg.cc/sxyCZmgB/Snap-Crab-No-Name-2022-5-12-2-35-56-No-00.png' border='0' alt='Snap-Crab-No-Name-2022-5-12-2-35-56-No-00'/></a>

**Comment by sy-b** [comment]
>  are allergic to some web browsers

idk but humans in general are afraid of 🐍. 😄

<br>

In that case, we'll probably have to make a minimalistic version.

Or, you can **continue using old versions**.

**Comment by Nightcaat** [comment]
Not very familiar with XP browsers but this might not be possible simply due to old age. In any case I find it unlikely that anyone will pick this up

**Comment by andika207** [comment]
> Not very familiar with XP browsers but this might not be possible simply due to old age. 

how many accounts do you have on github and why are you reviving this old thread just now ?
you remind me of this other one I was discussing yesterday with @NotLazy

**Comment by NotLazy** [comment]
> > Not very familiar with XP browsers but this might not be possible simply due to old age. 
> 
> 
> 
> how many accounts do you have on github and why are you reviving this old thread just now ?
> 
> you remind me of this other one I was discussing yesterday with @NotLazy
> 
> 

Why did you mention me?

**Comment by Nightcaat** [comment]
> how many accounts do you have on github and why are you reviving this old thread just now ?
This is my only GitHub account. Mostly just cleaning up issues since I have push access.

**Comment by andika207** [comment]
> > how many accounts do you have on github and why are you reviving this old thread just now ?
> 
> This is my only GitHub account. Mostly just cleaning up issues since I have push access.

aside from trolling you brought nothing new up to this issue. 

I saw a fox with blue ears so this got me thinking you had two accounts
